src/test/ui/traits/trait-alias-object.rs

```diff
@@ -1,9 +0,0 @@
-#![feature(trait_alias)]
-
-trait EqAlias = Eq;
```

Comment:
why was this file renamed? Most `ui` tests are failure tests, having `fail` in the name doesn't convey more information imo.

Reply:
Because there's now a run-pass test with this name in the same dir.